### ISSUE TITLE
"Guard not" is always shown, even if the guard is not negated

### ISSUE THREAD

**Issue by eregon** [Shopify/seafoam#6]
For instance on this picture it should be `Guard, else TranferToInterpreter`, I verified with `props` that it's `"negated": false,`:
![Screenshot from 2020-07-23 11-04-24](https://user-images.githubusercontent.com/168854/88284479-c5702d00-cced-11ea-84f5-0f6085f6e923.png)

**Comment by chrisseaton** [comment]
We can see the problem in `fib-ruby.bgv`:

```
% bundle exec bin/seafoam examples/fib-ruby.bgv:9:1248 props | grep negated
  "negated": true,
    "name_template": "FixedGuard(!={p#negated}) {p#reason/s}",
% bundle exec bin/seafoam examples/fib-ruby.bgv:9:2621 props | grep negated
  "negated": false,
    "name_template": "FixedGuard(!={p#negated}) {p#reason/s}",
```

<img width="1327" alt="Screenshot 2020-07-27 at 14 01 04" src="https://user-images.githubusercontent.com/341094/88544649-a1745a80-d011-11ea-9b62-227b68dba009.png">

It looks like we're confusing a symbol and string property. The idea of using both was that we could use strings for properties that appear actually in the original graph, and symbols for synthetic properties we use to annotate the graph, but maybe that's confusing.

But to fix - we need to look for the string not the symbol, and we can use this graph as a test.

**Comment by chrisseaton** [comment]
Fixed by #7.
